app/views/concerns/navigable_view.rb

```diff
@@ -131,7 +131,7 @@ def navigation_global_primary_nav_exhibitions_submenu_items
   end
 
   def navigation_global_primary_nav_blog_submenu_items
-    feed_items = feed_entry_nav_items(Cache::FeedJob::URLS[:blog][:all], 6)
+    feed_items = feed_entry_nav_items(Feed.find_by_slug('all-blog').url, 6)
```

Comment:
It is possible that this feed will not be found, e.g. if it gets deleted via the CMS. This code needs to handle the case when no record is found.